test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -345,10 +345,10 @@ Date: REGEX(.*)
     "requestSemaphoreWaitingTime": "REGEX(\d+.\d+)",
     "statisticsRequests": "3",
     "subCache": "REGEX([0-9a-f]{24})",
-    "subCacheInserts": "REGEX((2|3))",
+    "subCacheInserts": "REGEX((3|4))",
```

Comment:
A note about this "funny line here":  "subCacheInserts": "REGEX((3|4))"

When under valgrind, a subCacheRefresh may occur before this is measured and as (unfortunately) a subCacheRefresh uses the same Remove and Insert functions (and subCacheUpdate also), the statistics counters change because of the refresh.

This should probably be fixed when we change the statistics.